dkc/core/admin/folder.py

```diff
@@ -9,3 +10,12 @@ class FolderAdmin(admin.ModelAdmin):
     list_display_links = ['id', 'name']
 
     search_fields = ['name', 'id']
+
+    @transaction.atomic
+    def save_model(self, request, obj: Folder, form, change: bool):
+        if not change:
+            if obj.parent:
+                obj.tree = obj.parent.tree
+            else:
+                obj.tree = Tree.objects.create(quota=obj.creator.quota)
```

Comment:
@brianhelba it might make sense to grant admin permission to `obj.creator`, but I'll let you decide.

Reply:
I agree this makes more sense.